tests/regression/1_run_TA_API.sh

```diff
@@ -2,7 +2,7 @@ make
 
 redis-server &
 
-bazel run -- accelerator --ta_port=$1&
+bazel run //accelerator:accelerator_microhttpd -- --ta_port=$1 &
```

Comment:
Do we have to specify HTTP server framework explicitly?

Reply:
I'll change it to the original one since we've changed the framework to  microhttpd.